apel/db/records/summary.py

```diff
@@ -225,3 +227,53 @@ def get_ur(self):
         # We don't want the XML declaration, because the whole XML
         # document will be assembled by another part of the program.
         return doc.documentElement.toxml()
+
+
+class SummaryRecord04(SummaryRecord):
```

Comment:
Since #359 got closed. Added for @Sae126V . Should ensure to leverage the base class which we are extending.

~ Similarly for JobRecord04, NormalisedSummaryRecord04 class.